app/helpers/menu_helper.rb

```diff
@@ -83,4 +83,8 @@ def forcibly_denied_switch?(switch)
     account = current_account.provider? ? current_account : current_account.provider_account
     account.settings.public_send(switch).is_a?(Settings::SwitchDenied)
   end
+
+  def api_selector_services
+    @api_selector_services ||= site_account.provider? && current_user.present? ? current_user.accessible_services.includes(:account) : Service.none
```

Comment:
Would you care to explain why you do it like this @Martouta? I don't really like the way this method looks.   (`Service.none`?)

Reply:
ok @thomasmaas I will explain in a comment in the PR 😄

Reply:
Ok, I will write u in Slack 😄